napari/_qt/_tests/test_async_slicing.py

```diff
@@ -66,7 +67,7 @@ def test_async_slice_multiscale_image_on_pan(make_napari_viewer, qtbot, rng):
     # Check that we're initially slicing the middle of the first dimension
     # over the whole of lowest resolution image.
     assert viewer.dims.not_displayed == (0,)
-    assert viewer.dims.current_step[0] == 2
+    assert viewer.dims.current_step[0] == 1
```

Comment:
The multiscale tests are a little trickier because it's nice to have the high resolution data shape be even, so that's evenly split up in the low resolution data. So I just changed the expected initial `current_step`. Happy to make improvements here.